stages/test/test_bootc_install_to_fs.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+import os.path
+from contextlib import contextmanager
+from unittest.mock import patch
+
+from osbuild.testutil.imports import import_module_from_path
+
+
+@patch("subprocess.run")
+def test_bootc_instalL_to_fs(mock_run, tmp_path):
```

Comment:
What's up with the capital L?

Reply:
Nice catch and silly me, looks like I hit the "shift" key a bit too early when writing this :) Fixed!